packages/lesswrong/components/comments/CommentsItem/CommentsItem.jsx

```diff
@@ -138,7 +141,8 @@ class CommentsItem extends Component {
   }
 
   render() {
-    const { comment, currentUser, frontPage, nestingLevel=1 } = this.props
+    const { comment, currentUser, frontPage, nestingLevel=1, showPostTitle, classes } = this.props
+    const post = comment && comment.post || this.props.post
```

Comment:
It looks like you're doing this because this component is being used on different pages. Rather than reading the post from the comment, can you pass down "comment.post" as a prop in the caller? If that's not possible because of how your data is being loaded, consider splitting this into separate components for loading data vs rendering data.

Reply:
Sure, that might work. Will try it out.